rewards/utils/emission_utils.py

```diff
@@ -104,21 +104,19 @@ def fetch_setts(chain: str) -> List[str]:
     return list(filter(lambda x: x not in DISABLED_VAULTS, setts))
 
 
-def parse_schedules(schedules) -> Dict[str, List[Schedule]]:
-    """
-    Parse unlock shcedules
-    :param schedules: schedules to parse
-    """
+def get_schedules_by_token(schedules: List[Schedule]) -> Dict[str, List[Schedule]]:
     schedules_by_token = {}
-    console.log("Fetching schedules...")
     for schedule in schedules:
-        schedule = parse_schedule(schedule)
         if schedule.token not in schedules_by_token:
             schedules_by_token[schedule.token] = []
         schedules_by_token[schedule.token].append(schedule)
     return schedules_by_token
 
 
+def parse_schedules(schedules) -> List[Schedule]:
```

Comment:
Not so sure if you need a function just to eval list comprehension

Reply:
good catch, was an artifact of the functions we had before.